File: dojo/th.m/FizzBuzz/Java/2014-02-05 Extra/FizzBuzz/src/fizzbuzz/FizzBuzz.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package fizzbuzz;

/**
 *
 * @author T
 */
public class FizzBuzz {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        for(int i=1; i<=100; i++) {
            System.out.println("" + ((i%5==0 || i%3==0) ? ((i%3==0?"Fizz":"")+(i%5==0?"Buzz":"")) : i));
        }
    }
    
}

File: dojo/ad.p/FizzBuzz/JavaScript/2013-10-10/fizzbuzz.js
function FizzBuzz(max = 100) {
    for (var i = 1; i <= max; i++) {
        var out = '';
        if (i % 3 == 0) {
            out = 'Fizz';
        }
        if (i % 5 == 0) {
            out += 'Buzz';
        }
        console.log(out || i);
    }
}